util/src/harvey/cmd/build/paths.go

```diff
@@ -11,9 +11,6 @@ import (
 
 func init() {
 	harvey = os.Getenv("HARVEY")
-	if harvey != "" {
```

Comment:
The intent is this:

First we grab $HARVEY if it exists.
Then we try to get the root of the current tree from git. If that works (without error!) we use the result of git to overwrite what we read from $HARVEY
If we get past the git command and the harvey variable still isn't set, we admonish the user to set $HARVEY and bail out.

This code allows whatever git finds to supersede $HARVEY. We could do it the other way around, too.

I think it's acceptable to require people to set either HARVEY or PATH but not both...